omero/developers/Python.rst

```diff
@@ -310,6 +341,31 @@ Read data
                 well.getImage(index).getName(),\
                 well.getImage(index).getId()
 
+-  **List all annotations on an object. Get text from tags**
+
+::
+
+    for ann in project.listAnnotations():
+        print ann.getId(), ann.OMERO_TYPE,
+        print " added by ", ann.link.getDetails().getOwner().getOmeName()
+        if ann.OMERO_TYPE == omero.model.TagAnnotationI:
+            print "Tag value:", ann.getTextValue()
+
+-  **Get Links between Objects and Annotations**
+
+::
+
+    # Find Images linked to Annotation(s), move them to another Annotation
+    for link in conn.getAnnotationLinks('Image', ann_ids=[ann_id]):
+        print "Image:", link.getParent().name
+        # Update the underlying omero.model.ImageAnnotationLinkI.child
+        link._obj.child = omero.model.TagAnnotationI(new_tag_id False)
+        link.save()
+
+    # Find Annotations linked to Object(s), filtering by namespace optional
```

Comment:
"filtering by namespace optional" - probably optional in brackets ? The sentence starts with "Find", then continues with "filtering" (not "filter" , this would be more consistent)